Data/LeaderBoard.txt

```diff
@@ -0,0 +1,2 @@
+asdf 39
+bob 0
```

Comment:
ahh we have a problem here... the leaderboard file should not be saved in source control... this will be different for every player..... 

So we need to just remove the file from git ....  

Your program will just create the file if it does not exist... you may have to have put in some code to deal with trying to load the leaderboard the first time when the file does not exist.